components/ide/jetbrains/backend-plugin/src/main/kotlin/io/gitpod/jetbrains/remote/GitpodMetricControlProvider.kt

```diff
@@ -7,20 +7,62 @@ package io.gitpod.jetbrains.remote
 import com.jetbrains.ide.model.uiautomation.BeControl
 import com.jetbrains.rd.ui.bedsl.dsl.VerticalGridBuilder
 import com.jetbrains.rd.ui.bedsl.dsl.verticalGrid
+import com.jetbrains.rd.ui.bedsl.dsl.*
```

Comment:
This was suggested by IntelliJ but it looks a bit odd to me, having `.dsl.*` and other specific imports 🤔

Reply:
You can run optimize imports to resolve such things. Usually I don't care about import sections. Let IntellIJ does it thing.